packages/stark-core/package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "@nationalbankbelgium/stark-core",
   "version": "0.0.0-PLACEHOLDER-VERSION",
+  "module": "./esm5/stark-core.js",
+  "es2015": "./es2015/stark-core.js",
+  "main": "bundles/stark-core.umd.js",
+  "typings":"stark-core.d.ts",
```

Comment:
Maybe "types": "stark-core.d.ts" could be better than "typings" ?

Reply:
Indeed, "typings" was the previous name of this property, now it should be "types" (although both work fine)